prow/config.yaml

```diff
@@ -1834,6 +1926,35 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-e2e-gce
+    agent: kubernetes
+    context: pull-kubernetes-e2e-gce
+    rerun_command: "/test pull-kubernetes-e2e-gce"
+    trigger: "(?m)^/test (all|pull-kubernetes-e2e-gce),?(\\s+|$)"
+    always_run: true
+    # TODO(bentheelder): fork 1.10 after release
```

Comment:
?

Reply:
Heh. These are all copy + paste from the master jobs and then fixup the branch fields and image tags